docs/legacy/copied-from-beats/docs/shared-ssl-config.asciidoc

```diff
@@ -1,33 +1,14 @@
 [[configuration-ssl]]
-ifndef::apm-server[]
-== Configure SSL
-
-++++
-<titleabbrev>SSL</titleabbrev>
-++++
-endif::apm-server[]
-ifdef::apm-server[]
-== SSL output settings
-
-You can specify SSL options with any output that supports SSL, like {es}, {ls}, or Kafka.
-endif::[]
-
-ifndef::apm-server[]
-You can specify SSL options when you configure:
-
-* <<configuring-output,outputs>> that support SSL
-ifndef::no_dashboards[]
-* the <<setup-kibana-endpoint,{kib} endpoint>>
-endif::[]
-ifeval::["{beatname_lc}"=="heartbeat"]
-* <<configuration-heartbeat-options,{beatname_uc} monitors>> that support SSL
-endif::[]
-ifeval::["{beatname_lc}"=="metricbeat"]
-* <<metricbeat-modules,modules>> that define the host as an HTTP URL
-endif::[]
-endif::[]
-
-Example output config with SSL enabled:
+== SSL/TLS output settings
+
+****
+image:./binary-yes-fm-no.svg[supported deployment methods]
+
+These configuration options are only relevant to APM Server binary users.
```

Comment:
Should we point users to https://www.elastic.co/guide/en/fleet/current/fleet-settings.html#es-output-settings for configuring Elasticsearch TLS settings in Fleet?

Reply:
Absolutely! Thanks

Reply:
```suggestion
These configuration options are only relevant to APM Server binary users. Fleet-managed users should see the {fleet-guide}/fleet-settings.html[Fleet output settings].
```